packages/swapper/src/thorchain-utils/memo/assertAndProcessMemo.ts

```diff
@@ -225,6 +225,33 @@ export const assertAndProcessMemo = (memo: string): string => {
 
       return `${_action}:${basisPoints}:${THORCHAIN_AFFILIATE_NAME}:0`
     }
+    case 'tcy': {
+      // TCY:CLAIMADDR
+      const [_action, destAddr] = memo.split(':')
+
+      assertMemoHasDestAddr(destAddr, memo)
+
+      // Ensures we strip any additional bits which may not be supported for claim Txs
```

Comment:
There may or may not be support for affiliate name for tracking purposes, but we don't know that yet and the test cases don't seem to indicate that, see https://gitlab.com/-/snippets/4835159

Reply:
Doesn't seems for now according to https://gitlab.com/thorchain/thornode/-/merge_requests/4004/diffs#3a4ad8c2aca2115d159d22c2fc79fb82015203c2_264_164

Should we ask them?